samples/wifi/scan/src/main.c

```diff
@@ -332,8 +333,11 @@ int main(void)
 			}
 		}
 
+		net_bytes_from_str(params.mac_address.addr, sizeof(CONFIG_WIFI_MAC_ADDRESS),
```

Comment:
Please check the return value, I would also do some sanity checks on `CONFIG_WIFI_MAC_ADDRESS`